spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -445,6 +445,106 @@ definitions:
           type: u16
           desc: Issue of clock data
 
+ - MSG_EPHEMERIS_BDS:
+    id: 0x0089
+    short_desc: Satellite broadcast ephemeris for BDS
+    desc: |
+      The ephemeris message returns a set of satellite orbit
+      parameters that is used to calculate BDS satellite position,
+      velocity, and clock offset. Please see the BeiDou Navigation
+      Satellite System SIS-ICD Version 2.1, Table 5-9 for more details.
+    fields:
+      - common:
+          type: EphemerisCommonContent
+          desc: Values common for all ephemeris types
+      - tgd:
+          type: array
```

Comment:
Does the produced ephemeris use an array for these values as well? Or would it be clearer to break out the two values into distinct fields. I don't love using arrays to group items where distinct fields can be used.

Reply:
I'd prefer to break away from the array in the SBP message I think - in this instance it'll be much clearer and easier to understand

Reply:
The array has been removed.